packages/account/src/Sections/Security/Passwords/passwords-platform.tsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { useRequest } from '@deriv/api';
+import { CFD_PLATFORMS, getPlatformSettings } from '@deriv/shared';
+import { observer, useStore } from '@deriv/stores';
+import { localize } from '@deriv/translations';
+import FormSubHeader from 'Components/form-sub-header';
+import SentEmailModal from 'Components/sent-email-modal';
+import PlatformPartials from './platform-partials';
+
+type TPasswordsPlatformProps = {
+    has_dxtrade_accounts?: boolean;
+    has_mt5_accounts?: boolean;
+};
+
+/**
+ * Displays a change password instructions for MT5 and/or DXTrade.
+ * @name PasswordsPlatform
+ * @param [has_dxtrade_accounts=false] - Whether the user has DXTrade accounts.
+ * @param [has_mt5_accounts=false] - Whether the user has MT5 accounts.
+ * @returns {React.ReactNode}
+ */
+const PasswordsPlatform = observer(
+    ({ has_dxtrade_accounts = false, has_mt5_accounts = false }: TPasswordsPlatformProps) => {
+        const { mutate } = useRequest('verify_email');
+
+        const {
+            client: { email },
+        } = useStore();
+
+        const [identifier, setIdentifier] = React.useState('');
+        const [is_sent_email_modal_open, setIsSentEmailModalOpen] = React.useState(false);
+
+        const platform_name_dxtrade = getPlatformSettings('dxtrade').name;
+
+        const getPlatformTitle = () => {
+            let title = '';
+            if (has_mt5_accounts) {
+                title = localize('Deriv MT5 Password');
+            } else if (has_dxtrade_accounts) {
+                title = localize('{{platform_name_dxtrade}} Password', { platform_name_dxtrade });
+            }
+            return title;
+        };
+
+        const onClickSendEmail = (cfd_platform?: string) => {
+            const password_reset_code =
+                cfd_platform === CFD_PLATFORMS.MT5
+                    ? 'trading_platform_mt5_password_reset'
+                    : 'trading_platform_dxtrade_password_reset';
+
+            mutate({
+                payload: {
+                    verify_email: email,
+                    type: password_reset_code,
+
+                    url_parameters: {
+                        redirect_to: 3,
+                    },
+                },
+            });
+
+            setIdentifier(cfd_platform ?? '');
+            setIsSentEmailModalOpen(true);
+        };
+
+        return (
+            <React.Fragment>
+                <FormSubHeader title={getPlatformTitle()} />
+                <div className='account__passwords-wrapper'>
+                    {has_mt5_accounts && (
+                        <PlatformPartials
+                            type={CFD_PLATFORMS.MT5}
+                            handleClick={onClickSendEmail}
+                            description={localize(
```

Comment:
```suggestion
                            description={localize(
```
Again if it is possible (without too much refactoring), please use the <Localize /> component. Try to avoid using `localize()` as much as you can

Reply:
Will replace the localize function with component